### PR TITLE
Add support for alternate location of README file in exercises

### PR DESCRIPTION
**Pull request by eloots**
Apart from the default location of the README.md files in
`project/src/test/resources/README.md`, an alternate location can be
selected by setting `studentify.readme-files-in-exercise-root-folder`
to `true` in configuration

### CODE REVIEW

File: src/main/scala/com/lightbend/coursegentools/MasterSettings.scala [hunk 1]
```diff
@@ -63,6 +63,8 @@ class MasterSettings(masterRepo: File, optConfigurationFile: Option[String]) {
 
   val relativeSourceFolder: String = config.getString("studentify.relative-source-folder")
 
+  val readmeFilesInExerciseRootFolder: Boolean = config.getBoolean("studentify.readme-files-in-exercise-root-folder")
```
Comment (ennru) [inline src/main/scala/com/lightbend/coursegentools/MasterSettings.scala]: I'm not super-happy with the setting name. 
Maybe `readme-along-with-build-sbt` or `readme-in-root`, or define it the other way around like `readme-in-test-resources`?

File: sbtStudentCommands/Navigation.scala.template [hunk 2]
```diff
@@ -205,6 +213,8 @@ object Navigation {
         withZipFile(state, toProjectName) { () =>
           val src = new sbt.File(new sbt.File(Project.extract(state).structure.root), SSettings.studentifiedBaseFolder)
           val newExerciseSrc = new sbt.File(new sbt.File(Project.extract(state).structure.root), s".cue/$toProjectName")
+          copyExerciseReadme(newExerciseSrc, src)
+          println(s"~~> newExerciseSrc = ${newExerciseSrc} - src = $src")
```
Comment (ennru) [inline sbtStudentCommands/Navigation.scala.template]: Is the `println` intentional?

Comment (eloots) [reply sbtStudentCommands/Navigation.scala.template]: Oops - trailing debug code. Will remove it.

File: sbtStudentCommands/Navigation.scala.template [hunk 3]
```diff
@@ -137,7 +144,7 @@ object Navigation {
   }
 
   def gotoExerciseNr: Command = Command("gotoExerciseNr")(s => ExerciseNr) { (state, arg) =>
-    val ExerciseFmt = """.*exercise_([0-9][0-9][0-9])_\w+$""".r
+    val ExerciseFmt = """.*_([0-9][0-9][0-9])_\w+$""".r
```
[the remark this answers is not in the document]
Comment (eloots) [reply sbtStudentCommands/Navigation.scala.template]: Good point. Been thinking about this myself for while.

[the remark this answers is not in the document]
Comment (eloots) [reply sbtStudentCommands/Navigation.scala.template]: Probably need to spend some time to perform a general clean-up of the code.

**Comment by eloots** [comment]
Unrelated change: mark currently selected exercise in output of `listExercises` command:

```
man [e] > alpakka-primer > Coordinated shutdown > listExercises
  1.   step_001_HTTP_request
  2.   step_002_Extract_HTTP_entity
  3.   step_003_Parse_CSV
  4.   step_004_Producing_JSON
  5.   step_005_Cleanse_lines
  6. * step_006_Coordinated_shutdown
  7.   step_007_Almost_complete
  8.   step_008_Completed
```

**Pull request merged**